types/nodejs-captcha/nodejs-captcha-tests.ts

```diff
@@ -0,0 +1,12 @@
+import captcha from 'nodejs-captcha';
```

Comment:
A CommonJS import should be used here instead of ES6:

```suggestion
import captcha = require('nodejs-captcha');
```